tests/mcmc_test.py

```diff
@@ -130,9 +130,11 @@ def lp_f(x):
 @pytest.mark.parametrize(
     "method",
     (
-        "nuts",
-        "hmc",
-        "slice",
+        "nuts_pyro",
+        "hmc_pyro",
+        "nuts_pymc",
+        "hmc_pymc",
+        "slice_pymc",
```

Comment:
nice! so the tests run locally for me. I am a bit surprised though that there are the only tests. How about adding a very short tutorial showing how to use the PyMC samplers?